test/functional/rpc_blockchain.py

```diff
@@ -277,6 +277,9 @@ def _test_getdeploymentinfo(self):
         # calling with an explicit hash works
         self.check_signalling_deploymentinfo_result(self.nodes[0].getdeploymentinfo(gbci207["bestblockhash"]), gbci207["blocks"], gbci207["bestblockhash"], "started")
 
+        # restart clean with default activation heights
+        self.restart_node(0)
```

Comment:
Should this line be in the following commit?

Reply:
No, it's to make this commit pass the test on its own. This is because this sub-test restart the node with a different deployment height for Segwit, and its caller then calls `verifychain`. This would fail if we don't restart the node with the default deployment height because `ConnectBlock` would detect unexpected witnesses before activation height. This is essentially what the next commit checks explicitly.